sys/net/routing/nhdp/nhdp.c

```diff
@@ -337,8 +329,8 @@ static void write_packet(struct rfc5444_writer *wr __attribute__((unused)),
                          struct rfc5444_writer_target *iface __attribute__((unused)),
                          void *buffer, size_t length)
 {
-    ipv6_addr_t src, dst = IPV6_ADDR_ALL_NODES_LINK_LOCAL;
-    uint16_t sport, dport = MANET_PORT;
-    conn_udp_getlocaladdr(&conn, &src, &sport);
-    conn_udp_sendto(buffer, length, &src, sizeof(src), &dst, sizeof(dst), AF_INET, sport, dport);
+    sock_udp_ep_t ep = { .family = AF_INET6, .netif = SOCK_ADDR_ANY_NETIF,
+                         .port = MANET_PORT };
+    memcpy(ep.addr.ipv6, &ipv6_addr_all_nodes_link_local, sizeof(ep.addr.ipv6));
```

Comment:
its an array, so its arbitrary if you do `ep.addr.ipv6`, `&(ep.addr.ipv6)` or `&(ep.addr.ipv6[0])` I prefer the first.